src/components/patch/PatchContainer.test.tsx

```diff
@@ -1,21 +1,20 @@
 import { ExpansionPanel, InputBase, Select } from '@material-ui/core';
 import * as enzyme from "enzyme";
-import { PatchInfo } from 'evergreen.js/lib/models';
 import * as moment from 'moment';
 import * as React from "react";
 import * as InfiniteScroll from 'react-infinite-scroller';
 import * as rest from "../../rest/interface";
 import { Variant } from '../variant/Variant';
-import { Patch } from './Patch';
+import { Patch, PatchProps } from './Patch';
 import { PatchContainer } from "./PatchContainer";
 
 describe("PatchContainer", () => {
-
   const wrapper = enzyme.mount(<PatchContainer client={rest.EvergreenClient("", "", "", "", true)} username={"admin"} onFinishStateUpdate={null} />);
+
   const infiniteScroll = wrapper.find(InfiniteScroll);
-  infiniteScroll.prop("loadMore")(0);
+    infiniteScroll.prop("loadMore")(0);
```

Comment:
nit: was there a reason you made this whitespace change? I think it was fine before

Reply:
undid these changes but forget to push the commit